src/__test__/Home.test.jsx

```diff
@@ -0,0 +1,49 @@
+import {
```

Comment:
- [ ] As the previous reviewer mentioned Also You should add more unite tests and tests for all your components and pure functions in the app, kindly make sure to include this in your test folder